### PR TITLE
Reledmac doc corrections

### PR DESCRIPTION
**Pull request by tweh**
Hi, I made some corrections to the manuals and hope they can be merged this way …

### CODE REVIEW

File: reledmac.dtx [hunk 1]
```diff
@@ -373,7 +374,7 @@
 %
 % \subsection{Aim of the package}
 %
-% The \Eledmac{} package, together with LaTeX, provides several
+% The \eledmac{} package, together with \LaTeX, provides several
```
Comment (maieul) [inline reledmac.dtx]: to be corrected on \macpackage (I will do it)

File: reledmac.dtx [hunk 2]
```diff
@@ -388,7 +389,7 @@
 % \end{itemize}
 %
 %
-% \Eledmac{} allows the scholar engaged in preparing a critical
+% \eledmac{} allows the scholar engaged in preparing a critical
```
Comment (maieul) [inline reledmac.dtx]: \Macpackage (with upercase)

File: reledmac.dtx [hunk 3]
```diff
@@ -752,7 +753,7 @@
 % Its content will be printed before the beginning of \protect\cs{pstart} / after the end of  \protect\cs{pend} instead of the argument of \protect\cs{AtEveryPstart} / \protect\cs{AtEveryPend}.
 % If you need to start a \protect\cs{pstart} by brackets, or to add brackets after a \protect\cs{pend}, just add a \protect\cs{relax} between \protect\cs{pstart}/\protect\cs{pend} and the brackets.
 %
-% This feature is also useful when typesetting verses (see~\reff{verse}) or \protect\package{eledpar} (see~\reff{eledpar}).
+% This feature is also useful when typesetting verses (see~\reff{verse}) or \protect\package{reledpar} (see~\reff{eledpar}).
```
Comment (maieul) [inline reledmac.dtx]: to be change to \parpackage.

File: reledmac.dtx [hunk 4]
```diff
@@ -954,7 +956,7 @@
 % sub-line numbering. One use for line-number locking is in printing
 % poetry: there the line numbers should be those of verse lines
 % rather than of printed lines, even when a verse line requires several
-% printed lines.
+% printed lines. But in this case you may use the \verb|\stanza| mechanism, see \reff{verse}.
```
Comment (maieul) [inline reledmac.dtx]: \cs{stanza} (I will do)

File: reledmac.dtx [hunk 5]
```diff
@@ -1039,14 +1041,14 @@
 %
 % \Macpackage manages many series of notes of each category.
 % A series of notes is identified by an uppercase letter.
-% When the series letter is at the beginning of a command name, it refers to a critical footnote.
-% When the series letter is at the end of a command name, it refers to a critical endnote.
+% When the series letter is at the \emph{beginning} of a command name, it refers to a critical note.
```
Comment (maieul) [inline reledmac.dtx]: no, footnote, because there is also critical endnotes.

File: reledmac.dtx [hunk 6]
```diff
@@ -1039,14 +1041,14 @@
...
+% When the series letter is at the \emph{end} of a command name, it refers to a familiar note.
```
Comment (maieul) [inline reledmac.dtx]: also footnote (we could imagine familiar endnote)

File: reledmac.dtx [hunk 7]
```diff
@@ -1166,21 +1168,21 @@
 % \DescribeMacro{\Eendnote}
 % The package also maintains five separate series of endnotes.
 %
-% If you do not need the endnotes facility, you should use \verb+noend+ option when loading \protect\package{eledmac}.
+% If you do not need the endnotes facility, you should use \verb+noend+ option when loading \protect\package{reledmac}.
```
Comment (maieul) [inline reledmac.dtx]: to be change in \macpackage.

File: reledmac.dtx [hunk 8]
```diff
@@ -1252,19 +1254,20 @@
 % \DescribeMacro{\linenum}
 % \label{linenum}
 % You can use \protect\cs{linenum}\marg{arg} to change the line numbers passed to
-% the notes. The notes are actually given seven parameters: the page,
+% the notes. \meta{arg} actually constist of seven parameters: the page,
```
Comment (maieul) [inline reledmac.dtx]: consist.

File: reledmac.dtx [hunk 9]
```diff
@@ -1353,7 +1356,7 @@
 %               it is all}%
 %       }{\lemma{\sameword{sw} \ldots all}\Afootnote{critical note}}.%
 % \end{verbatim}
-% \protect\package{eledmac} cannot know if the ``sw'' in \protect\cs{lemma} refers to the word after ``thing'', after ``other'', or after ``again''.
+% \protect\package{reledmac} cannot know if the ``sw'' in \protect\cs{lemma} refers to the word after ``thing'', after ``other'', or after ``again''.
```
Comment (maieul) [inline reledmac.dtx]: \macpackage.

File: reledmac.dtx [hunk 10]
```diff
@@ -1450,7 +1453,7 @@
 % sequential footnote
 % marks in the text can be separated by 
 % commas\textsuperscript{3,4} like so.
-% As a convenience \Eledmac{} provides this automatically.
+% As a convenience \reledmac{} provides this automatically.
```
Comment (maieul) [inline reledmac.dtx]: \macpackage\

File: reledmac.dtx [hunk 11]
```diff
@@ -1539,19 +1542,19 @@
...
 %
-% The notes arrangement must be called after having defined the document geometry setting. If you must change geometry setting inside your document, do not forget to call again not arrangement.
+% The notes arrangement must be called after having defined the document geometry setting. If you must change geometry setting inside your document, do not forget to call again note arrangement.
 % 
 % \protect\cs{hsize} has been set for the pages that use
 % this series of notes; otherwise \TeX\ will try to put too
 % many or too few of these notes on each page. If you need to
 % change the \protect\cs{hsize} within the document, call
-% \protect\cs{footparagraph} again afterwards to take account of the new
+% the arrangement macor again afterwards to take account of the new
```
Comment (maieul) [inline reledmac.dtx]: mafro

File: reledmac.dtx [hunk 12]
```diff
@@ -1568,7 +1571,7 @@
 % \changes{v1.20.0}{2015/03/22}{Add \protect\cs{Xtwolines} and \protect\cs{Xmorethantwolines} hooks}
 % \label{Xtwolines}
 % \DescribeMacro{\Xtwolines}\DescribeMacro{\Xmorethantwolines}
-% If a lemma is printed on two subsequent lines, \Eledmac\ will print the first and the last line numbers. 
+% If a lemma is printed on two subsequent lines, \reledmac\ will print the first and the last line numbers.
```
Comment (maieul) [inline reledmac.dtx]: \macpackage

File: reledmac.dtx [hunk 13]
```diff
@@ -1734,9 +1737,9 @@
 % \changes{v1.22.0}{2015/04/25}{Add option for lemma separator inside endnotes}
 % \DescribeMacro{\Xendlemmaseparator}
 % By default, there is no separator inside endnotes between the lemma and the content of the note. You can use \protect\cs{Xendlemmaseparator}\oarg{s}\marg{Xendlemmaseparator} to change this. The optional argument can be used to specify the series in which it is used. 
-% An common value of \verb+<Xendlemmaseparator>+ is \verb+\rbracket+.
+% A common value of \meta{Xendlemmaseparator} is \verb+\rbracket+.
```
Comment (maieul) [inline reledmac.dtx]: \cs{\rbracket}

File: reledmac.dtx [hunk 14]
```diff
@@ -1828,12 +1833,12 @@
 % \verb+<code>+ must be one of the following command:
 % \begin{description}
 %  \item[\protect\cs{justifying}] to have text justified, as usual with \LaTeX. You can also let \verb+<code>+ empty.
-%  \item[\protect\cs{raggedright}] to have text left aligned, but \emph{without hyphenation}. That is the default \protect\package{eledmac} setting. 
-%  \item[\protect\cs{RaggedRight}] to have text left aligned \emph{with hyphenation}.
+%  \item[\protect\cs{raggedright}] to have text left aligned, but \emph{without hyphenation}. That is the default \protect\package{reledmac} setting.
```
Comment (maieul) [inline reledmac.dtx]: \reledmac

File: reledmac.dtx [hunk 15]
```diff
@@ -1828,12 +1833,12 @@
 % \verb+<code>+ must be one of the following command:
 % \begin{description}
 %  \item[\protect\cs{justifying}] to have text justified, as usual with \LaTeX. You can also let \verb+<code>+ empty.
-%  \item[\protect\cs{raggedright}] to have text left aligned, but \emph{without hyphenation}. That is the default \protect\package{eledmac} setting. 
-%  \item[\protect\cs{RaggedRight}] to have text left aligned \emph{with hyphenation}.
+%  \item[\protect\cs{raggedright}] to have text left aligned, but \emph{without hyphenation}. That is the default \protect\package{reledmac} setting. 
+%  \item[\protect\cs{RaggedRight}] to have text left aligned \emph{with hyphenation} (requires \package{ragged2e}).
 %  \item[\protect\cs{raggedleft}] to have text right aligned, but \emph{without hyphenation}.
-%  \item[\protect\cs{RaggedLeft}] to have text right aligned \emph{with hyphenation}.
+%  \item[\protect\cs{RaggedLeft}] to have text right aligned \emph{with hyphenation} (requires \package{ragged2e}).
```
Comment (maieul) [inline reledmac.dtx]: ragged2e

File: reledmac.dtx [hunk 16]
```diff
@@ -1865,10 +1870,10 @@
 % \subsubsection{Ragging}
 % \changes{v1.12.0}{2014/08/05}{New options for ragged-paragraph notes}
 % \DescribeMacro{\Xragged}
-% Text in paragraphed critical notes is justified, but you can use \protect\cs{Xragged}\oarg{s}+{L}+ if you want it to be ragged left, or \protect\cs{Xragged}\oarg{s}+{R} if you want it to be ragged right.
+% Text in paragraphed critical notes is justified, but you can use \protect\cs{Xragged}\oarg{s}\verb+{L}+ if you want it to be ragged left, or \protect\cs{Xragged}\oarg{s}\verb+{R}+ if you want it to be ragged right.
```
Comment (maieul) [inline reledmac.dtx]: \arg

File: reledmac.dtx [hunk 17]
```diff
@@ -1865,10 +1870,10 @@
...
 
 % \DescribeMacro{\raggedX}
-% Text in paragraphed footnotes is justified, but you can use \protect\cs{raggedX}\oarg{s}+{L}+ if you want it to be ragged left, or \protect\cs{raggedX}\oarg{s}+{R} if you want it to be ragged right.
+% Text in paragraphed footnotes is justified, but you can use \protect\cs{raggedX}\oarg{s}\verb+{L}+ if you want it to be ragged left, or \protect\cs{raggedX}\oarg{s}\verb+{R}+ if you want it to be ragged right.
```
Comment (maieul) [inline reledmac.dtx]: \arg

File: reledmac.dtx [hunk 18]
```diff
@@ -1881,13 +1886,13 @@
 % You can change the vertical space printed before the rule of the critical notes with \protect\cs{Xbeforenotes}\oarg{s}\marg{l}.
 % The default value is \verb|1.2em plus .6em minus .6em|.
 %
-% \textbf{Be careful, the standard \LaTeX\ footnote rule, which is used by \protect\package{eledmac}, decreases by 3pt. This 3pt decrease is not changed by this command..}
-% \DescribeMacro{\beforenotesX}
+% \textbf{Be careful, the standard \LaTeX\ footnote rule, which is used by \protect\package{reledmac}, decreases by 3pt. This 3pt decrease is not changed by this command..}
```
Comment (maieul) [inline reledmac.dtx]: \macpackage

File: reledmac.dtx [hunk 19]
```diff
@@ -1881,13 +1886,13 @@
...
 %
+% \DescribeMacro{\beforenotesX}
 % You can change the vertical space printed before the rule of the familiar notes with \protect\cs{beforenotesX}\oarg{s}\marg{l}.
 % The default value is \verb|1.2em plus .6em minus .6em|.
 %
-% \textbf{Be careful, the standard \LaTeX\ footnote rule, which is used by \protect\package{eledmac}, decreases 3pt. These 3pt are not changed by this command.}
+% \textbf{Be careful, the standard \LaTeX\ footnote rule, which is used by \protect\package{reledmac}, decreases 3pt. These 3pt are not changed by this command.}
```
Comment (maieul) [inline reledmac.dtx]: \macpackage

File: reledmac.dtx [hunk 20]
```diff
@@ -1922,14 +1915,14 @@
 % You can change the vertical space printed after the rule of the critical notes with \protect\cs{Xafterrule}\oarg{s}\marg{l}.
 % The default value is \verb|0pt|.
 %
-% \textbf{Be careful, the standard \LaTeX\ footnote rule, which is used by \protect\package{eledmac}, adds 2.6pt. These 2.6pt are not changed by this command.}
+% \textbf{Be careful, the standard \LaTeX\ footnote rule, which is used by \protect\package{reledmac}, adds 2.6pt. These 2.6pt are not changed by this command.}
```
Comment (maieul) [inline reledmac.dtx]: \macpackage

File: reledmac.dtx [hunk 21]
```diff
@@ -1900,20 +1905,8 @@
 % \DescribeMacro{\prenotesX}
 % You can want the space before the first printed (in a page) series of familiar notes not to be the same as before other series. Default value is \verb+0pt+.
 % You can do it with \protect\cs{prenotesX}\marg{l}.
-% You can disable this feature by setting the length to 0~pt.
```
Comment (maieul) [inline reledmac.dtx]: Why deleting?

File: reledmac.dtx [hunk 22]
```diff
@@ -1900,20 +1905,8 @@
...
-%
-% \DescribeMacro{\preXnotes}
-% You can set the space before the first series of critical notes printed on each page
-% and set a different amount of space for subsequent the series on the page.
-% You can do it with \protect\cs{preXnotes}\marg{l}. Default value is \verb+0pt+.
 % You can disable this feature by setting the length to 0pt.
 %
-%
```
Comment (maieul) [inline reledmac.dtx]: When deleting ?

File: reledmac.dtx [hunk 23]
```diff
@@ -1922,14 +1915,14 @@
...
+% \textbf{Be careful, the standard \LaTeX\ footnote rule, which is used by \protect\package{reledmac}, adds 2.6pt. These 2.6pt are not changed by this command.}
 %
 %
 % \DescribeMacro{\afterruleX}
-% You can change the vertical space printed after the rule of the familiar notes with \protect\cs{beforenotesX}\oarg{s}\marg{l}.
+% You can change the vertical space printed after the rule of the familiar notes with \protect\cs{afterruleX}\oarg{s}\marg{l}.
 % % The default value is \verb|0pt|.
 %
-% \textbf{Be careful, the standard \LaTeX\ footnote rule, which is used by \protect\package{eledmac}, adds 2.6pt. These 2.6pt are not changed by this command.}
+% \textbf{Be careful, the standard \LaTeX\ footnote rule, which is used by \protect\package{reledmac}, adds 2.6pt. These 2.6pt are not changed by this command.}
```
Comment (maieul) [inline reledmac.dtx]: \macpackage

File: reledmac.dtx [hunk 24]
```diff
@@ -1937,19 +1930,19 @@
...
 % Be careful with the two previous commands. 
 % Actually, for technical purposes, one paragraphed note is considered as one block. 
-% Consequently, it ca not be broken between two pages, even if you used these commands. 
+% Consequently, it can not be broken between two pages, even if you used these commands. 
 % The debug is in the todolist.
 % \subsection{Footnotes and the \parpackage columns} 
 % \changes{v1.13.0}{2014/09/16}{Add \protect\cs{Xnoteswidthliketwocolumns} and \protect\cs{noteswidthliketwocolumnsX}}
 % \DescribeMacro{Xnoteswidthliketwocolumns}\DescribeMacro{noteswidthliketwocolumnsX}
-% If you use \protect\package{eledpar} \protect\cs{columns} macro, you can call :
+% If you use \protect\package{reledpar} \protect\cs{columns} macro, you can call :
```
Comment (maieul) [inline reledmac.dtx]: \parpackage

File: reledmac.dtx [hunk 25]
```diff
@@ -2062,7 +2055,7 @@
 % \DescribeMacro{\stanza}
 % \DescribeMacro{\&}
 % Use \verb?\stanza? at the start of a stanza. Each line in a stanza
-% is ended by an ampersand (\&), and the stanza itself is ended
+% is ended by an ampersand (\verb+&+), and the stanza itself is ended
```
Comment (maieul) [inline reledmac.dtx]: \expandafter\cs\expandafter{\&} (or something like it, to be tested)

File: reledmac.dtx [hunk 26]
```diff
@@ -2086,7 +2079,7 @@
 % gives the hanging indentation to be used if the stanza line requires more
 % than one print line.
 %  
-%  If it is known that each stanza line will fit on more than one print line,
+%  If it is known that each stanza line will fit in one print line,
```
Comment (maieul) [inline reledmac.dtx]: why deleting more than?

File: reledmac.dtx [hunk 27]
```diff
@@ -2975,8 +2963,8 @@
 % \subsection{Prevent page break in a long verses}
 %\label{lednopbinversetrue}\DescribeMacro{\lednopbinversetrue} You can also decide to prevent page breaks between two lines of a long verse. To do this, use \verb+nopbinverse+ when loading package, or add \protect\cs{lednopbinversetrue} in the beginning of your file (better: in your preamble). 
 %
-%This feature works only with verse of 2 lines, not more. It works at the third run, or at fourth run with \eledpar.
-% By default, when a long verse runs normally between two pages, a page break will be placed at the beginning of the verse. However, if you have added\verb+ledpbsetting{after}+, the page break will be placed at the end of the long verse, and the page containing the long verse will have one extra line.
+%This feature works only with verse of 2 lines, not more. It works at the third run, or at fourth run with \reledpar.
```
Comment (maieul) [inline reledmac.dtx]: \parpackage

File: reledmac.dtx [hunk 28]
```diff
@@ -2975,8 +2963,8 @@
...
+% By default, when a long verse runs normally between two pages, a page break will be placed at the beginning of the verse. However, if you have added \verb+\ledpbsetting{after}+, the page break will be placed at the end of the long verse, and the page containing the long verse will have one extra line.
```
Comment (maieul) [inline reledmac.dtx]: \cs{ledpbsetting}\arg{after}

**Comment by maieul** [comment]
I have made some annotation for new correction. I will do the merge and implement them later.

**Comment by tweh** [comment]
Hi, do I have to do something more now? Im not sure … :-)

**Comment by maieul** [comment]
no, it should be ok
-----
Ma�eul Rouquette
www.maieul.net
http://geekographie.maieul.net/

Le 18 ao�t 2015 � 10:21, Tobias Weh <notifications@github.com> a �crit :

**Pull request merged**